### DISCUSSION TITLE
canvas didn't display when app starts

### DISCUSSION THREAD

**Discussion by chj-damon** [iddan/react-native-canvas#248] [Q&A]
I'm using react-native-canvas in my app, but it seems something went wrong but I don't know what caused it.

I prepared this repository to reproduce this issue:
https://github.com/chj-damon/rn-canvas-issue-reproduce

the step to reproduce my problem is:
1. npm install
2. npx react-native run-ios (run-android)
you can see that canvas didn't display on the screen after the app starts.
3. modify anything in app.js, then you'll see that canvas appears.

Why didn't it show for the first time, but have to wait for any changes?

**Comment by chj-damon** [comment]
![image](https://user-images.githubusercontent.com/4902684/72975539-f38eb600-3e0b-11ea-9eec-e50ea91914ae.png)

![image](https://user-images.githubusercontent.com/4902684/72975481-d4902400-3e0b-11ea-8ed5-f07925bd8558.png)

**Comment by gousen-pride** [comment]
I`m also having this issue. On IOS 12 works, but not on 13.x.
On android everything seems to be working as it should.

**Comment by kem226** [comment]
I am also having this issue. I have seen it on web as well. Android works fine. Anyone know a workaround for now? a way to trigger the re-load in app?

**Comment by CodeAnnihilator** [comment]
I also had this issue, so made PR here to fix it https://github.com/iddan/react-native-canvas/pull/175

**Comment by iddan** [comment]
Merged the PR. Will make a release soon.
